src/components/Form.js

```diff
@@ -123,19 +124,39 @@ export default class Form extends Component {
     return null;
   }
 
+  getUsedFormData = formData => {
+    const fields = this.formElement.querySelectorAll("[name]");
+    const inputNames = Object.keys(fields).reduce((filtered, i) => {
```

Comment:
I think it would be simpler to replace this with 
```
fields.map(e => e.name).filter(e => e.name)
```

Reply:
You could actually just put

```
fields.map(e => e.name)
```

Because the lodash pick function doesn't mind if you pass in null/undefined arguments in the array.

Reply:
I think i can simplify, but unfortunately `querySelectorAll` returns a `NodeList` instead of an array so I can't use map.